docs/tutorials/python/profiler.md

```diff
@@ -265,10 +262,10 @@ Please note that: to be able to see the previously described information, you ne
 
 ```python 
 # Set profile_all to True
-profiler.set_config(profile_all=True, aggregate_stats=True, continuous_dump = True)
+profiler.set_config(profile_all=True, aggregate_stats=True, continuous_dump=True)
```

Comment:
generally the "=" is supposed to be separated by 1 space.

Reply:
I changed this to be consistent with the sections above which does not have spaces around "="